docs/rst/use_cases/vulcanexus_cloud.rst

```diff
@@ -0,0 +1,150 @@
+.. include:: ../exports/alias.include
+
+.. _vulcanexus_cloud:
+
+################
+Vulcanexus Cloud
+################
+
+*Vulcanexus* is an extended *ROS 2* distribution provided by *eProsima* which includes additional tools for an improved
+user and development experience, such as *Fast-DDS-Monitor* for monitoring the health of DDS networks, or *micro-ROS*,
+a framework aimed at optimizing and extending *ROS 2* toolkit to microcontroller-based devices.
+
+Apart from plain LAN-to-LAN communication, Cloud environments such as container-oriented platforms have also been
+present throughout the |ddsrouter| design phase. In this walk-through example, we will set up both a *Kubernetes*
+(|k8s|) network and local environment in order to establish communication between a pair of ROS nodes, one sending
+messages from a LAN (talker) and another one (listener) receiving them in the Cloud. This will be accomplished by having
+a |ddsrouter| instance at each side of the communication.
+
+.. figure:: /rst/figures/ddsrouter_overview_wan.png
```

Comment:
Nice picture :+1: 
I would suggest to use it in order to explain the whole scenario that is being explained in this section. Something fast like "we have a local ... and want to communicte with a cloud k8s cluster ..."